caluma/caluma_core/serializers.py

```diff
@@ -71,7 +71,6 @@ def create(self, validated_data):
         user = self.context["request"].user
         validated_data["created_by_user"] = user.username
         validated_data["created_by_group"] = user.group
-
```

Comment:
Was this line removed by black?

Reply:
Hm, not sure, but no need to touch that file. fixed :)